projects/wc/src/app/components/generic-ui/list-view/delete-resource-confirmation-modal/delete-resource-modal.component.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  OnInit,
+  inject,
+  input,
+  output,
+  viewChild,
+} from '@angular/core';
+import {
+  FormBuilder,
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+} from '@angular/forms';
+import {
+  FieldDefinition,
+  Resource,
+  ResourceNodeContext,
+} from '@openmfp/portal-ui-lib';
+import {
+  BarComponent,
+  DialogComponent,
+  IconComponent,
+  InputComponent,
+  TextComponent,
+  TitleComponent,
+  ToolbarButtonComponent,
+  ToolbarComponent,
+} from '@ui5/webcomponents-ngx';
+
+@Component({
+  selector: 'delete-resource-modal',
+  standalone: true,
+  imports: [
+    ReactiveFormsModule,
+    DialogComponent,
+    TitleComponent,
+    ToolbarButtonComponent,
+    ToolbarComponent,
+    InputComponent,
+    BarComponent,
+    IconComponent,
+    TextComponent,
+  ],
+  templateUrl: './delete-resource-modal.component.html',
+  styleUrl: './delete-resource-modal.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DeleteResourceModalComponent implements OnInit {
+  fields = input<FieldDefinition[]>([]);
+  context = input<ResourceNodeContext>();
+  resource = output<Resource>();
+  dialog = viewChild<DialogComponent>('dialog');
+
+  cdr = inject(ChangeDetectorRef);
+
+  innerResource: Resource;
```

Comment:
I am aware that this one might come from the copy paste implemntation, but I see no need for the innerResource which is one -to-ont with the resource ? and thus we shouldn't need to use the cdr = inject(ChangeDetectorRef); it's all based on signals (or should be)